src/resources/formats/html/esbuild-analysis-cache.json

```diff
@@ -0,0 +1,30 @@
+{
+  "quarto.js": {
+    "inputs": {
+      "quarto.js": {
+        "bytes": 26327,
+        "imports": [],
+        "format": "esm"
+      }
+    },
+    "outputs": {
+      "../../../../../../../../../../dev/null": {
```

Comment:
Is this expected to have this fake path here ? 

Probably but I want to confirm as it could surprise future us or new developer later in a few times. 😅

Reply:
Yes, that path is currently irrelevant for us. I should probably add a post-processing pass that canonicalizes it.

Reply:
Fixed by [ca10597](https://github.com/quarto-dev/quarto-cli/pull/11821/commits/ca10597f9916e4cc7a97ea2100641f338ead0aff)